packages/react-topology/README.md

```diff
@@ -29,11 +29,99 @@ or
 npm install @patternfly/react-topology --save
 ```
 
-## Usage
+## Basic Usage
 
-It's strongly advised to use the PatternFly Base CSS in your whole project, or some components may diverge in appearance:
+To use React Topology out-of-the-box, you will first need to have a data model. This data model should process your back-end data to return nodes and edges.
```

Comment:
@jeff-phillips-18 do you have any ideas on how we can improve the semantics here a bit?

Reply:
I think we want to get a couple of points across:

- To display nodes, edges and groups in the topology graph, you will need to transform your back-end data into a `Model` (maybe link this to the `Model` in types).
- These model objects contain the information needed to display the nodes and edges. Each node and edge has a set of properties used by PF Topology as well as a `data` field which can be used to customize the nodes and edges by the application.